vscode-web/main.tf

```diff
@@ -97,6 +97,30 @@ variable "settings" {
   default     = {}
 }
 
+variable "offline" {
+  type        = bool
+  description = "Just run code-server in the background, don't fetch it from GitHub"
```

Comment:
We seem to have a few different terms (`VS Code Web`, `VS Code Server`, and `code-server`) in here.  I think maybe `VS Code Web` might make the most sense given the title of the module.

Reply:
Ok. Will look.

Reply:
Just FYI, I only put a comment on this line, but there are other references to `code-server` that we should also update (six, if I counted right, seven if we want to also update the `Microsoft Visual Studio Code Server` line).  Sorry if I was unclear!